installation/proc-install-rhdh-helm.adoc

```diff
@@ -27,17 +27,34 @@ For more information about creating a project in OpenShift, see link:https://doc
 . From the *Developer Catalog* panel, click *Helm Chart*.
 . Search for _Developer Hub_ in the search bar and select the *{product}* card.
 . Click *Create*.
-. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.*, and adjust the other values if needed.
+. Use one of the following views to edit the Helm configuration:
 
-. Alternatively, copy the OpenShift router host (for example: `apps.<clusterName>.com`) to `global.clusterRouterBase`, and adjust other values if needed, such as `global.clusterRouterBase: apps.example.com`. 
```

Comment:
PR #135 duplicates these changes

Reply:
@linfraze, If you look at the [SME comment above ](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/139#pullrequestreview-1976857517)he asked to include the YAMl code snippet in the doc for better visualization. this is why i have included YAMl code snippet as well.